src/server/worker/master.go

```diff
@@ -558,7 +558,8 @@ func (a *APIServer) waitJob(pachClient *client.APIClient, jobInfo *pps.JobInfo,
 				if _, err := col.NewSTM(ctx, a.etcdClient, func(stm col.STM) error {
 					chunks := a.chunks(jobID).ReadWrite(stm)
```

Comment:
For consistency, we should probably change this to chunksCol or plansCol to plans.